readthedocs/templates/robots.txt

```diff
@@ -0,0 +1,7 @@
+User-agent: *
+{% for path in hidden_paths %}
+Disallow: {{ path }}
+{% empty %}
+Disallow:
```

Comment:
Previously this was `Allow: /`, but that is just a convention, the standard way of allowing everything is with  `Disallow:` https://www.robotstxt.org/orig.html#format